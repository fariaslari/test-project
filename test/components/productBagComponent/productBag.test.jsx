import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import ProductBag from './../../../src/components/productBagComponent/ProductBag';

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


describe('ProductBag', () => {
  it('should render correctly', () => {

    const output = shallow(
        <ProductBag />
    );
    
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});