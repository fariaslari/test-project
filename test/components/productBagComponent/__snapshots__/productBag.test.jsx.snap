// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductBag should render correctly 1`] = `
<div
  className="productBag"
>
  <div
    className="title"
  >
    <img
      src="/assets/img/bag-icon.png"
    />
    <span>
      sacola
    </span>
    <div
      className="productCount"
    >
      <span>
        0
      </span>
    </div>
  </div>
  <div
    className="emptyBag"
  >
    Sua sacola est√° vazia
  </div>
</div>
`;
